doc/scan.rst

```diff
@@ -556,6 +576,11 @@ Policies
             | One of the :ref:`POLICY_REPLICA` values such as :data:`aerospike.POLICY_REPLICA_MASTER`
             |
             | Default: ``aerospike.POLICY_REPLICA_SEQUENCE``
+        * **ttl** (:class:`int`)
+            | The default time-to-live (expiration) of the record in seconds. This field will only be
+            | used on background scan writes if :py:attr:`aerospike.Scan.ttl` is set to :data:`aerospike.TTL_CLIENT_DEFAULT`.
+            |
+            | There are also special values that can be set for this field. See :ref:`TTL_CONSTANTS`.
```

Comment:
This is the scan policy, right? Shouldn't this say something like "this policy will only be used if :py:attr:`aerospike.Scan.ttl` is NOT set to :data:`aerospike.TTL_CLIENT_DEFAULT`"?

Reply:
The scan policy's ttl field is only supposed to be set in the default scan policy in the client config, i.e in `config['policies']['scan']['ttl']` where `config` is passed to `aerospike.client(config)`. It's not meant to be set in a scan policy that is passed to a function call as an argument. I can clarify this with a doc change